pkg/driver/efs_watch_dog.go

```diff
@@ -37,14 +45,10 @@ logging_file_count = 10
 state_file_dir_mode = 750
 
 [mount]
-dns_name_format = {fs_id}.efs.{region}.{dns_name_suffix}
+dns_name_format = {az}.{fs_id}.efs.{region}.{dns_name_suffix}
 dns_name_suffix = amazonaws.com
 #The region of the file system when mounting from on-premises or cross region.
-{{if .Region -}}
-region = {{.Region -}}
```

Comment:
@anonymint will fargate work without this?

Reply:
My bad, I should probably put that back for fargate support

Reply:
yea I assume taht if both region and az are provided then az mount option takes precedence?

Reply:
If both region and az are provided, then efs-utils will use both to resolve dns (see line 48)

Reply:
I see, so someone could provide a nonsense combination . anyway, I think as long as we keep this here then the fargate situation will still work . IIRC the fargate issue is that there is no instance metadata for efs-utils to figure out what region it's in to construct the efs dns name, so we have to explicitly provide it, that's all...

Reply:
There is always a chance that someone can provide a bad combination of az and region. If someone provided a bad combination, efs-utils will fail to resolve dns and throw a mount failure exception.

Reply:
@wongma7 and @kbasv yes Fargate needs that line, probably too late it's back already, all good then.